server/config/routes/scripts.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+
+import { saveNewScript,
+         getAllScripts,
+         getScriptById,
+         updateScriptById,
+         addQuestionToScript,
+         getQuestionsByScriptId,
+         deleteScriptById } from '../../controllers/scripts';
+
+const router = express.Router();
+
+router.route('/').post(saveNewScript);
+router.route('/').get(getAllScripts);
+router.route('/:id').get(getScriptById);
+router.route('/:id').put(updateScriptById);
+router.route('/:id').delete(deleteScriptById);
+router.route('/addQuestionToScript/:id/:questionId/:sequenceNum').post(addQuestionToScript);
```

Comment:
`router.route('/:id/scriptQuestions/:questionId/:sequenceNum').post(addQuestionToScript);
router.route('/:id/scriptQuestions/').get(getQuestionsByScriptId);`

like that?